glue/__init__.py

```diff
@@ -118,6 +118,11 @@ def make_selector_func(roi):
     from .qt.custom_viewer import CustomViewer
     return CustomViewer.create_new_subclass(name, **kwargs)
 
+# Load built-in plugins
+from .config import load_plugins
+load_plugins()
```

Comment:
Hmm this will end up importing things like Qt immediately. We've always tried to keep glue.core importable without Qt (although maybe this isn't a useful restriction)